src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -17,17 +17,19 @@
 <kd-help-section>
   <md-input-container class="md-block" md-is-error="ctrl.isNameError()">
     <label>App name</label>
-    <input ng-model="ctrl.name" name="name" namespace="ctrl.namespace" required kd-unique-name
-        ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 500, 'blur': 0 } }">
-    <md-progress-linear class="kd-deploy-form-progress" md-mode="indeterminate"
-        ng-show="ctrl.form.name.$pending">
-    </md-progress-linear>
-    <ng-messages for="ctrl.form.name.$error" role="alert" multiple>
-      <ng-message when="required">Application name is required.</ng-message>
-      <ng-message when="uniqueName">
-        Application with this name already exists within namespace <i>{{ctrl.namespace}}</i>.
-      </ng-message>
-    </ng-messages>
+    <input ng-model="ctrl.name"
+           name="name"
```

Comment:
ng-model-options were here to reflect name change not on every keystroke. When you removed this, the help text  "
Ports are optional. If specified, a Service will be created mapping the Port (incoming) to a target Port seen by the container. The internal DNS name for this Service will be" updates on every keystroke. This is distracting. With model options this updates on blur or after you stop typing, which is better. Revert? What do you think? @maciaszczykm